os/booting-on-digitalocean.md

```diff
@@ -7,6 +7,8 @@ schedules per channel. You can [disable this feature][reboot-docs], although we
 don't recommend it. Read the [release notes][release-notes] for specific
 features and bug fixes.
 
+**NOTE: Deploying CoreOS on DigitalOcean is by standard not secure,** because DigitalOcean does not allow private networking. That means that even though you use private networking, you can still be attacked from others inside DigitalOceans private network.
```

Comment:
Yes, that is good, but the last line is more like:
**to prevent others from ~~reading your traffic~~ controlling your cluster**.